packages/fxa-auth-server/scripts/transfer-users/apple.js

```diff
@@ -168,18 +164,24 @@ export class AppleUser {
     // link the Apple account to the FxA account.
     try {
       if (this.mock) {
-        console.log(`Mock: No user found, creating new user with email: ${appleEmail}`);
-        this.setSuccess({uid: uuid.v4({}, Buffer.alloc(16)).toString('hex'), email: appleEmail});
+        console.log(`Mock: No user found, creating new user with email: ${pocketEmail}`);
+        this.setSuccess({uid: uuid.v4({}, Buffer.alloc(16)).toString('hex'), email: pocketEmail});
         return;
       }
       
       const emailCode = await random.hex(16);
       const authSalt = await random.hex(32);
       const [kA, wrapWrapKb] = await random.hex(32, 32);
+      
+      // Is Pocket email private? Use the new private email from Mozilla Apple.
+      let accountEmail = pocketEmail;
+      if (!pocketEmail || pocketEmail.indexOf('privaterelay.appleid.com') > -1) {
```

Comment:
Not sure why CodeQL resolved my previous comment, so here we go again:

Instead of `.indexOf() > -1` not sure if we can use [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) or even a fan favorite [`.endsWith()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith). And if this is meant to be an email domain, any objections to changing it to include the "@" sign for a bit of extra safety?

Plus whatever the CodeQL error is above. Not sure if it's saying we should be validating the email address' hostname versus something like peter+privaterelay.appleid.com@gmail.com.

Reply:
I do like .endsWith.